backend/app/seed/data/toeic_lr.py

```
@@ -0,0 +1,148 @@
+            "part5",
+            n,
+            30,
+            "Choose the word or phrase that best completes the sentence.",
+        )
+    )
+    n += 30
+
+    qs.extend(_sets("part6", n, 4, 4, prefix="text", passage_line="Text"))
+    n += 16
+
+    # 54 reading-comprehension items: 10×3 + 6×4 = 54
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)